analyzere/base_resources.py

```diff
@@ -116,6 +116,9 @@ def to_dict(value):
 
 class AnalyzeReObject(object):
     def __init__(self, **kwargs):
+        type_value = kwargs.get('_type', None)
```

Comment:
do we want to pop `_type` or do we still need to keep a copy of the reference with the underscore for compatibility?

Reply:
good question...

Reply:
pop doesn't seem to have any negative effect, so I'll use it instead. thanks